datasync/chaindatafetcher/kas/contract_caller.go

```diff
@@ -92,8 +99,7 @@ func (f *contractCaller) supportsInterface(contract common.Address, opts *bind.C
 	// case 2: the contract does not implements fallback function
 	// - the call can be reverted: returns "evm: execution reverted"
 	isSupported, err := caller.SupportsInterface(opts, interfaceID)
-	if err != nil && (strings.Contains(err.Error(), "abi: unmarshalling empty output") ||
-		strings.Contains(err.Error(), "evm: execution reverted")) {
+	if err != nil && (strings.Contains(err.Error(), errMsgEmptyOutput) || strings.Contains(err.Error(), errMsgEvmReverted)) {
```

Comment:
How about making a function to help other people to understand the logic here? e.g., `msgEmptyOrReverted`

Reply:
Added a extra method here 6fe22020166b796a401889c395243c8ecaba73b4.